features/a_user_can_navigate_pages_using_side_and_navbar.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Navigate the website
+  As a user
+  In order to navigate the website
+  I would like to be able to click on different buttons
+
+Background: 
+    Given I visit the landing page
+
+Scenario: User can navigate different pages by using buttons
+    When I click "Products" button
+    Then I should be on product page
+    When I click "Login" button
+    Then I should be on login page
+    When I visit the landing page
+    And I click "Signup" button
+    Then I should be on signup page
+    When I visit the landing page
```

Comment:
Are you moving to another page using Capybara? Then it should be a new scenario... Not sure what you are trying to accomplish here

Reply:
When user click on login, our navbar is not showing on the devise view, hence, it's not finding the other links. In this case, the signup button. We'll fix it when we start modifying devise views